src/led.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <stdint.h>
+#include <libopencm3/stm32/gpio.h>
+
+#define LED_M0_R GPIO6
+#define LED_M0_B GPIO7
+#define LED_M1_R GPIO8
+#define LED_M1_B GPIO9
```

Comment:
question: M0 / M1 as in schematic or silkscreen?

Reply:
The silkscreen